src/mongo/editors/MongoFindOneResultEditor.ts

```diff
@@ -18,6 +18,9 @@ export class MongoFindOneResultEditor implements ICosmosEditor<IMongoDocument> {
     constructor(databaseNode: MongoDatabaseTreeItem, collectionName: string, data: string) {
         this._databaseNode = databaseNode;
         this._collectionName = collectionName;
+        if (data === "null") {
```

Comment:
I don't understand this check. Is it really checking for the _string_ null? When can this happen?

Reply:
Yes. The variable name `data` is misleading - `data` is a _string_  of the query result (returned from `executeCommand`), and hence line 25 has a `EJSON.parse`. 

It is the string "null" when there are no documents matching the filter. :)

Reply:
I didn't see the need to rename the variable since the type annotations explain the expected handling.

Reply:
Suggest adding a comment. A string of "null" is very uncommon, plus this repo doesn't have strictNullChecks on so the types aren't super accurate. Someone could easily mess this up if they didn't know the full story. Just add something like:
```
// Intentionally checking against string of "null", which can be returned by mongo shell
```